.circleci/config.yml

```diff
@@ -28,11 +28,15 @@ jobs:
       - run:
           name: Fusion standardization
           command: ./scripts/run_in_ci.sh Rscript analyses/fusion_filtering/01-fusion-standardization.R -f "data/pbta-fusion-arriba.tsv.gz" -c "arriba" -o "scratch/arriba.tsv" 
- 
+
       - run:
           name: Fusion general filtering
-          command: ./scripts/run_in_ci.sh Rscript analyses/fusion_filtering/02-fusion-filtering.R -S scratch/arriba.tsv -e data/pbta-gene-expression-rsem-fpkm.polya.rds -r -a "GTEx|HGNC_GENEFAM|DGD_PARALOGS|Normal|BodyMap|ConjoinG" -j 1 -s 10 -i "in-frame|frameshift|other" -R analyses/fusion_filtering/references/ -o scratch/arriba -t 1                         
-     
+          command: ./scripts/run_in_ci.sh Rscript analyses/fusion_filtering/02-fusion-filtering.R -S scratch/arriba.tsv -e data/pbta-gene-expression-rsem-fpkm.polya.rds -r -a "GTEx|HGNC_GENEFAM|DGD_PARALOGS|Normal|BodyMap|ConjoinG" -j 1 -s 10 -i "in-frame|frameshift|other" -R analyses/fusion_filtering/references/ -o scratch/arriba -t 1
+
+      - run:
+          name: Fusion annotation
```

Comment:
```suggestion
          name: Fusion Annotation
```
Testing if the branch allows write (trying to help @jaclyn-taroni update to master)

Reply:
![Screen Shot 2019-10-22 at 3 26 44 PM](https://user-images.githubusercontent.com/542643/67323371-5fe1dc80-f4e0-11e9-8082-483fa75573cc.png)

I'm pretty sure this is why @jaclyn-taroni can't update to master.

Reply:
Feel free to ignore this suggestion 😁